pallets/task-scheduler/src/tasks/rpc.rs

```diff
@@ -0,0 +1,45 @@
+use alloc::string::String;
+//#[allow(dead_code)]
+#[derive(serde::Deserialize, serde::Serialize, Clone, Debug)]
+pub struct JsonRpcError {
+	pub code: i32,
+	pub message: String,
+}
+
+pub(super) mod error {
```

Comment:
Do RPC specific errors belong in the pallet or should they remain in `creditcoin`?

Reply:
That's arguable, I'm in favor of moving everything not related to Creditcoin's model or logic out of it. When in doubt, we leave where it is.